demo/index.html

```diff
@@ -9,35 +9,124 @@
-    <link href="../../paper-styles/demo-pages.html" rel="import">
-    <link rel="import" href="simple-fit.html">
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
 
-    <style>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="simple-fit.html">
+  <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+  <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
 
-      .sized {
-        max-width: 50%;
-        max-height: 50%;
-        padding: 20px;
+  <style is="custom-style" include="demo-pages-shared-styles">
+    demo-snippet {
+      --demo-snippet-code: {
+        max-height: 250px;
       }
+    }
+  </style>
 
-    </style>
+</head>
 
-  </head>
-  <body unresolved>
+<body unresolved class="centered">
+  <h3>
+    An element with <code>IronFitBehavior</code> can be centered in
+    <code>fitInto</code> or positioned around a <code>positionTarget</code>
+  </h3>
+  <demo-snippet>
+    <template>
+      <style>
+        .target {
+          cursor: pointer;
+          text-align: center;
+          display: inline-block;
+          box-sizing: border-box;
+          border: 1px solid;
+          width: 100px;
+          padding: 20px 0;
+          margin: 5px;
+        }
 
-    <simple-fit id="myFit" auto-fit-on-attach class="sized" onclick="myFit.refit()">
-        Centered in window.<br><br>
-        Resize the window and click here to refit.
-    </simple-fit>
+        #myFit {
+          z-index: 10;
+          padding: 20px;
+          overflow: auto;
+          width: 220px;
+        }
+      </style>
+      <template is="dom-bind">
+        <template is="dom-repeat" items="[[containers]]">
+          <div class="target" on-tap="updatePositionTarget">Target</div>
+        </template>
+        <simple-fit id="myFit" auto-fit-on-attach>
+          <h2>Align</h2>
+          <p>
+            <button on-tap="updateAlign" vertical-align="top">top</button>
+            <button on-tap="updateAlign" vertical-align="bottom">bottom</button>
+            <button on-tap="updateAlign" vertical-align="auto">auto</button>
+            <button on-tap="updateAlign" vertical-align>null</button>
+          </p>
+          <p>
+            <button on-tap="updateAlign" horizontal-align="left">left</button>
+            <button on-tap="updateAlign" horizontal-align="right">right</button>
+            <button on-tap="updateAlign" horizontal-align="auto">auto</button>
+            <button on-tap="updateAlign" horizontal-align>null</button>
+          </p>
+          <button on-tap="toggleNoOverlap">Toggle overlap</button>
+        </simple-fit>
+        <script>
+          var defaultTarget = Polymer.dom(myFit).parentNode;
+          var t = document.querySelector('template[is="dom-bind"]');
```

Comment:
`t` -> `template` - I don't want to scroll back up the page just to figure out what a `t` is 😜